src/_main.ts

```diff
@@ -41,6 +41,8 @@ async function startUpdater() {
 
 /** Registers the commands, starts the bots and the updater. */
 async function start() {
+  const modeName = process.env.NODE_ENV || 'unknown';
+  botLogger.info(`Starting main in ${modeName} mode`);
```

Comment:
This is missing the `Main` label for consistency. I should probably add a loggable interface to make this less annoying.

Reply:
Fixed. 

Another approach would be to instantiate a logger for each class with a set label, and then always use `this.logger()`